### PR TITLE
BaSP-TG-52: bug fixed of filter method

### PR DESCRIPTION
**Pull request by NicolasFontana**
Now the variable be into the URL like the others endpoints.

### CODE REVIEW

File: src/tests/super-admins.test.js [hunk 1]
```diff
@@ -33,30 +33,6 @@ describe('GET superAdmin by ID', () => {
   });
 });
 
-describe('GET superAdmins by filters', () => {
```
Comment (Javier-Cervera) [inline src/tests/super-admins.test.js:36]: I think we should write some tests, at least one successful and one unsuccessful `getFilteredSuperAdminBy****`

File: src/controllers/super-admins.js [hunk 2]
```diff
@@ -46,33 +46,160 @@ const createSuperadmin = async (req, res) => {
...
+    const superAdminByEmail = await models.SuperAdmin.find({ email: req.params.email });
+    if (superAdminByEmail.length !== 0) {
+      return res.status(200).json({
+        message: 'Superadmins filtered by email',
+        data: superAdminByEmail,
+        error: false,
+      });
+    }
+    return res.status(404).json({
+      message: 'Super admin not found',
```
Comment (Javier-Cervera) [inline src/controllers/super-admins.js:68]: We could return a message like: `` `The Super Admin with **** ${req.params.****} has not been found` `` for more clarification for the user

File: src/controllers/super-admins.js [hunk 3]
```diff
@@ -46,33 +46,160 @@ const createSuperadmin = async (req, res) => {
...
+  } catch (error) {
+    return res.status(400).json({
+      message: error.message,
+      data: {},
+      error: true,
+    });
+  }
+};
+
+// SEARCH SUPERADMINS BY PASSWORD
```
Comment (agusting19) [inline src/controllers/super-admins.js:145]: why superadmins has a filter by password? this make sense?

File: src/tests/super-admins.test.js (hunk 1 above)
Comment (agusting19) [reply src/tests/super-admins.test.js:36]: i agree with Javi

**Review by agusting19 — APPROVED** [review]

File: src/tests/super-admins.test.js (hunk 1 above)
Comment (Javier-Cervera) [reply src/tests/super-admins.test.js:36]: Done

**Review by Javier-Cervera — APPROVED** [review]
Good job!

**Pull request merged**